src/components/SafeListSidebar/SafeList/SafeListItem.tsx

```diff
@@ -95,17 +117,19 @@ const SafeListItem = ({
     <ListItem button onClick={handleOpenSafe} ref={safeRef}>
       <StyledIcon type="check" size="md" color="primary" checked={isCurrentSafe} />
       <StyledPrefixedEthHashInfo hash={address} name={safeName} shortName={shortName} showAvatar shortenHash={4} />
-      <ListItemSecondaryAction>
+      <StyledListItemSecondaryAction>
         {ethBalance ? (
-          `${formatAmount(ethBalance)} ${nativeCurrencySymbol}`
+          <StyledText size="lg">
+            {formatAmount(ethBalance)} {nativeCurrencySymbol}
+          </StyledText>
         ) : showAddSafeLink ? (
-          <Link to={generateSafeRoute(LOAD_SPECIFIC_SAFE_ROUTE, routesSlug)} onClick={handleLoadSafe}>
-            <Text size="sm" color="primary">
+          <StyledButton onClick={handleLoadSafe} size="md" variant="outlined">
```

Comment:
If the button takes you to a different page, it should be a link and not a button to be semantically correct

Reply:
I agree! When calling `handleLoadSafe` we also close the sidebar and switch the network if necessary so I think in this case it is ok to do the routing manually within the onClick handler